apps/web/modules/settings/my-account/components/CompanyEmailOrganizationBanner.tsx

```diff
@@ -0,0 +1,84 @@
+"use client";
+
+import { useRouter } from "next/navigation";
+
+import { useFlagMap } from "@calcom/features/flags/context/provider";
+import { useLocale } from "@calcom/lib/hooks/useLocale";
+import { Button } from "@calcom/ui/components/button";
+
+import { MailIcon } from "./MailIcon";
+
+type CompanyEmailOrganizationBannerProps = {
+  onDismissAction: () => void;
+};
+
+export const CompanyEmailOrganizationBanner = ({ onDismissAction }: CompanyEmailOrganizationBannerProps) => {
+  const { t } = useLocale();
+  const router = useRouter();
+  const flags = useFlagMap();
+
+  const handleLearnMore = () => {
+    const redirectPath = flags["onboarding-v3"]
+      ? "/onboarding/organization/details"
+      : "/settings/organizations/new";
+    router.push(redirectPath);
+  };
+
+  if (!flags["onboarding-v3"]) {
+    return null;
+  }
+
+  return (
+    <div className="border-subtle from-muted relative mb-6 overflow-hidden rounded-lg border bg-gradient-to-r to-transparent p-4">
+      {/* Dot grid background with fade */}
+      <svg className="pointer-events-none absolute inset-0 h-full w-full" xmlns="http://www.w3.org/2000/svg">
+        <defs>
+          {/* Dot pattern - 12px spacing */}
+          <pattern id="banner-dot-pattern" x="0" y="0" width="12" height="12" patternUnits="userSpaceOnUse">
+            <circle cx="1" cy="1" r="0.8" fill="currentColor" className="text-subtle" opacity="0.3" />
+          </pattern>
+          {/* Fade mask from left to right */}
+          <linearGradient id="banner-fade-gradient" x1="0%" y1="0%" x2="100%" y2="0%">
+            <stop offset="0%" stopColor="white" stopOpacity="1" />
+            <stop offset="10%" stopColor="white" stopOpacity="0.4" />
+            <stop offset="100%" stopColor="white" stopOpacity="0" />
+          </linearGradient>
+          <mask id="banner-fade-mask">
+            <rect x="0" y="0" width="100%" height="100%" fill="url(#banner-fade-gradient)" />
+          </mask>
```

Comment:
I had troubles with clip masks so ended up going svg approach for this. much easier